jobs/JGEFS_ATMOS_AVGSPR_GEMPAK_META

```diff
@@ -122,8 +122,6 @@ $HOMEgefs/scripts/exgefs_avgspr_gempak_meta.sh.ecf
 #############################################################
 err=$?
 
-cat $pgmout
```

Comment:
I'd like to keep this statement, because in the future, if any failure happened, we can see the content of the file

Reply:
$pgmout isn't defined as anything.